mk/submodule.mk

```diff
@@ -0,0 +1,21 @@
+# Copy from the Makefile of libtuv to support different platforms
+UNAME_M := $(shell uname -m)
+UNAME_S := $(shell uname -s)
```

Comment:
Add `FIXME` to mention the limitation of supported operating system listing.

Reply:
Umm......
I'm not sure what you expect to see.
Like listing the operating system that `dcurl` supports but `libtuv` does not or vice versa?